manifests/crane/params.pp

```diff
@@ -6,6 +6,6 @@
   $debug = false
   $port = 5000
 
-  $data_dir = '/var/lib/crane/metadata'
+  $data_dir = '/var/lib/pulp/published/docker/v2/app'
```

Comment:
Well we had this value before puppet-crane got deprecated. Look at => https://github.com/Katello/puppet-crane/blob/master/manifests/params.pp#L8 

Dont see why we need to override ?